cli/tests/environment-activate.bats

```diff
@@ -495,3 +495,41 @@ env_is_activated() {
   assert_success
   assert_line "emacs"
 }
+
+
+# ---------------------------------------------------------------------------- #
+
+# bats test_tags=activate:scripts:on-activate
+@test "'hook.on-activate' runs" {
+  "$FLOX_BIN" delete -f
+  "$FLOX_BIN" init
+  "$FLOX_BIN" edit -f "$BATS_TEST_DIRNAME/activate/on-activate.toml"
+  # Run a command that causes the activation scripts to run without putting us
+  # in the interactive shell
+  run "$FLOX_BIN" activate -- echo "hello"
+  # The on-activate script creates a directory whose name is the value of the
+  # "$foo" environment variable.
+  run [ -d "$PROJECT_DIR/bar" ]
+  assert_success
```

Comment:
nonblocking, see https://bats-core.readthedocs.io/en/stable/writing-tests.html#when-not-to-use-run
```suggestion
  [ -d "$PROJECT_DIR/bar" ]
```

Reply:
Fixed